src/types/currency.ts

```diff
@@ -47,13 +48,13 @@ export type CollateralIdLiteral =
     | CurrencyIdLiteral.KINT
     | CurrencyIdLiteral.INTR;
 
-export const CollateralAmount = [PolkadotAmount, KusamaAmount];
+export const CollateralAmount = [PolkadotAmount, KusamaAmount, KintsugiAmount];
```

Comment:
Suggestion: I think in the mid-term it would be beneficial to get the collateral currencies at runtime from the parachain itself. Otherwise, for any new added collateral asset, the lib needs to be updated instead of us being able to deploy collateral assets without lib/UI upgrades.

Reply:
A bit late to reply (my apologies for that), but: 100% agreed.
Would be nice if we could generate the types from chain specs.

Reply:
Ok, then let's do that as a follow-up PR?